docs/content/latest/api/ysql/datatypes/type_array/functions-operators/string-to-array.md

```diff
@@ -0,0 +1,115 @@
+---
+title: string_to_array()
+linkTitle: string_to_array()
+headerTitle: string_to_array()
+description: string_to_array()
+menu:
+  latest:
+    identifier: string-to-array
+    parent: array-functions-operators
+isTocNested: false
+showAsideToc: false
+---
+**Purpose:** return a one-dimensional `text[]` array by splitting the input `text` value into subvalues using the specified `text` value as the delimiter. Optionally allows a specified `text` value to be interpreted as `null`.
```

Comment:
```suggestion
**Purpose:** return a one-dimensional `text[]` array by splitting the input `text` value into subvalues using the specified `text` value as the delimiter. Optionally, allows a specified `text` value to be interpreted as `null`.
```

Reply:
Eh? You forgot to make "return" start with "R". See what I mean now. I have to to the global search for **Purpose:** anyway. So this 30 second per change (yes, I timed it) saved nothing and cost us each a lot.